libraries/resource_hab_sup_systemd.rb

```diff
@@ -63,4 +63,4 @@ class HabSupSystemd < HabSup
       end
     end
   end
-end
+end unless system('hab sup status')
```

Comment:
You def don't want to shell out with system in a cookbook

Reply:
Understood. do you have thoughts on a better guard? we will likely have to do this on every service resource

Reply:
can you explain exactly what the problem you're trying to solve for is?

Reply:
When running this cookbook in effortless. the supervisor resources are failing every converge after the initial. It's looking like the version pinning is the what's causing the issue.

Reply:
confirmed.. version pinnings were all mismatched. causes the sup and service resources to get a little crazy. I'd like to find a way to set them to always latest and let the customer manger their versions